fabfile.py

```diff
@@ -89,7 +87,6 @@ def update():
         python_dependency(repo.name, PYTHON_VERSION)
 
     python_dependency('heltour', PYTHON_VERSION)
```

Comment:
@lakinwecker Is it necessary to have this and the above "container" at the same time? Also not entirely sure what purpose these serve

Reply:
@mathgeek These jam a `.pth` file into the virtual environment which allows python to find the modules appropriately.

Reply:
And yes, they're required.